_episodes_rmd/08-plot-ggplot2.Rmd

```diff
@@ -334,11 +348,11 @@ elements. The x-axis is too cluttered, and the y axis should read
 "Life expectancy", rather than the column name in the data frame.
 
```

Comment:
Not sure how the below edits happened since they didn't show up in my initial comparison. Regardless, they're just spaces at the end of the line and can be ignored.